scripts/00_deploy-all.js

```diff
@@ -0,0 +1,118 @@
+require('dotenv').config();
+const web3 = require('web3');
+const pricefeeds = require('./helpers/chainlink-pricefeed-addresses');
+
+module.exports = async function deployAll(options) {
+    const hardhat = require('hardhat');
+    await hardhat.run('compile');
+
+    // if (process.env.HARDHAT_NETWORK !== options.network) hardhat.changeNetwork(options.network);
+
+    const accounts = await hardhat.ethers.getSigners();
+    const deployer = accounts[0];
+    const protocol = accounts[1];
+    const network = hardhat.network.name;
+    let CLpricefeed = pricefeeds[network];
+    console.log(`Deploying all to ${network}...`);
+    const observerAddress = process.env.OBSERVER_ADDRESS;
```

Comment:
Is this mandatory even to deploy to localhost / hardhat? maybe it should only be mandatory for the real networks